flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/table/UnsignedTypeConversionITCase.java

```diff
@@ -43,136 +44,81 @@
+    private static final String user = "root";
+    private static final String password = "";
+
+    private static final Map<String, String> DEFAULT_CONTAINER_ENV_MAP =
+            new HashMap<String, String>() {
+                {
+                    put("MYSQL_ROOT_HOST", "%");
+                }
+            };
+
+    public static final MySQLContainer<?> MYSQL_CONTAINER =
+            new MySQLContainer<>(MYSQL_57_IMAGE)
+                    .withEnv(DEFAULT_CONTAINER_ENV_MAP)
+                    .withUsername(user)
+                    .withPassword(password)
+                    .withLogConsumer(new Slf4jLogConsumer(logger));
+
+    private static String baseUrl;
+    private static Connection connection;
     private StreamTableEnvironment tEnv;
 
     @BeforeClass
-    public static void prepareMariaDB() throws IllegalStateException {
-        boolean initDbSuccess = false;
-        int i = 0;
-        // The initialization of maria db instance is a little unstable according to past CI tests.
-        // Add retry logic here to avoid initialization failure.
-        while (i < INITIALIZE_DB_MAX_RETRY) {
-            try {
-                db = DB.newEmbeddedDB(DBConfigurationBuilder.newBuilder().build());
-                db.start();
-                dbUrl = db.getConfiguration().getURL(DEFAULT_DB_NAME);
-                connection = DriverManager.getConnection(dbUrl);
-                try (Statement statement = connection.createStatement()) {
-                    statement.execute("CREATE DATABASE IF NOT EXISTS `" + DEFAULT_DB_NAME + "`;");
-                    ResultSet resultSet =
-                            statement.executeQuery(
-                                    "SELECT SCHEMA_NAME FROM "
-                                            + "INFORMATION_SCHEMA.SCHEMATA WHERE SCHEMA_NAME = '"
-                                            + DEFAULT_DB_NAME
-                                            + "';");
-                    if (resultSet.next()) {
-                        String dbName = resultSet.getString(1);
-                        initDbSuccess = DEFAULT_DB_NAME.equalsIgnoreCase(dbName);
-                    }
-                }
-            } catch (Exception e) {
-                logger.warn("Initialize DB failed.", e);
-                stopDb();
-            }
-            if (initDbSuccess) {
-                break;
+    public static void launchContainer() throws Exception {
+        MYSQL_CONTAINER.start();
+        baseUrl =
+                MYSQL_CONTAINER
+                        .getJdbcUrl()
+                        .substring(0, MYSQL_CONTAINER.getJdbcUrl().lastIndexOf("/"));
+        connection = DriverManager.getConnection(baseUrl, user, password);
+        try (PreparedStatement ps =
+                connection.prepareStatement(
+                        "CREATE DATABASE IF NOT EXISTS `" + DEFAULT_DB_NAME + "`;")) {
+            ps.execute();
+            connection.close();
+            connection = null;
```

Comment:
why this statement, given in the next one you overwrite the connection variable anyway?